File: app/Reserva.php
<?php

namespace App;

use Illuminate\Database\Eloquent\Model;

class Reserva extends Model
{
    protected $fillable = ['status', 'cpf', 'preco', 'espetaculo', 'poltrona'];

    public function espetaculo()
    {
        return $this->belongsTo('App\Espetaculo', 'espetaculo');
    }

    public function poltrona()
    {
        return $this->belongsTo('App\Poltrona', 'poltrona');
    }
}

File: app/models/Reserva.py
from django.db import models
from app.models.Espetaculo import Espetaculo
from app.models.Poltrona import Poltrona

class Reserva(models.Model):
    
    espetaculo = models.ForeignKey(Espetaculo,on_delete = models.CASCADE, related_name = 'reservas')
    poltrona = models.ForeignKey(Poltrona, on_delete = models.CASCADE, related_name = 'reservas')
    


